src/zope/browserresource/tests/test_i18nfile.py

```diff
@@ -37,10 +37,33 @@
 from zope.i18n.interfaces import INegotiator
 from zope.i18n.negotiator import negotiator
 
-from zope.i18n.tests.testii18naware import TestII18nAware
 
 test_directory = os.path.dirname(p.__file__)
 
+class AbstractTestII18nAwareMixin(object):
+
+
+    def setUp(self):
```

Comment:
Too many blank lines here for my taste (and I'm sure flake8 would agree with me ;).

Reply:
That's a result of copy/paste from a diff. Honestly, I did notice the extra lines before I committed...but I really just wanted to see if you would comment on them 😉